### ISSUE TITLE
error while querying vote proposal endpoint

### ISSUE THREAD

**Issue by msobh13** [Zondax/namadexer#99]
error while querying vote proposal endpoint  /tx/vote_proposal/1   or any proposal number 

`{"Error":"Database error: error occurred while decoding column \"vote\": mismatched types; Rust type `bool` (as SQL type `BOOL`) is not compatible with SQL type `TEXT`"}`

**Comment by gnosed** [comment]
As a follow-up to this. It also appears while indexing. It happened on the last version of the shielded expedition at block height 7625.

```
  2024-02-05T21:22:20.357439Z ERROR namadexer::indexer: Closing block producer task due to an error saving last block: Database error: error returned from database: column "vote" is of type boolean but expression is of type text
    at src/indexer/mod.rs:229

Error: DB(Database(PgDatabaseError { severity: Error, code: "42804", message: "column \"vote\" is of type boolean but expression is of type text", detail: None, hint: Some("You will need to rewrite or cast the
expression."), position: Some(Original(254)), where: None, schema: None, table: None, column: None, data_type: None, constraint: None, file: Some("parse_target.c"), line: Some(587), routine: Some("transformAssignedExpr") }))
Makefile:49: recipe for target 'run_indexer' failed
make: *** [run_indexer] Error 1
```

**Comment by dimiandre** [comment]
should be fixed now

**Comment by msobh13** [comment]
Still not fixed

**Comment by rllola** [comment]
@msobh13 Are you sure you upgraded ? 
The change is visible on `main` (see https://github.com/Zondax/namadexer/blob/main/src/server/tx.rs#L211)

Let me know if you still have it.

**Comment by msobh13** [comment]
I am using different commits so many be the one i am using didnot have the fix .i will check again . Thanks

**Comment by rllola** [comment]
Yes sorry about that. I just released a new tag v0.1.0-rc.5 that should be stable.

**Comment by msobh13** [comment]
Thanks Lola
